lib/msal-node/test/client/ManagedIdentitySources/MachineLearning.spec.ts

```diff
@@ -106,42 +107,6 @@ describe("Acquires a token successfully via an Machine Learning Managed Identity
             );
         });
 
-        test("acquires a User Assigned Resource Id token", async () => {
```

Comment:
Is the behavior in this removed test covered by the new test? Or was this testing a formerly successful path that now throws the error?